cores/arduino/stm32/PeripheralPins.h

```diff
@@ -71,5 +71,7 @@ extern const PinMap PinMap_Ethernet[];
 //*** QUADSPI ***
 extern const PinMap PinMap_QUADSPI[];
 
+//*** WKUP ***
```

Comment:
This should not be here. Only one linked to Peripheral <--> pins should be here.